### PR TITLE
Client: Sequence diagram for DataBridge

### PR DESCRIPTION
**Pull request by addshore**

### CODE REVIEW

File: systems/WikibaseClient/06-Runtime_View.md [hunk 1]
```diff
@@ -2,7 +2,43 @@
...
+
+Once the Data Bridge is open the User can edit supported values.
+
+```mermaid
+sequenceDiagram
+    autonumber
+    participant User
+    participant Page as Wikitext-generated content
+    participant Bridge as Data Bridge
+    User->>+Bridge: Enters new values, clickes save
```
Comment (tzhelyazkova) [inline systems/WikibaseClient/06-Runtime_View.md:34]:
```suggestion
    User->>+Bridge: Enters new values, chooses an update strategy, clickes save
```

**Review by tzhelyazkova — COMMENTED** [review]
Looks great! The diagram is bootiful.
I made one small suggestion below.

**Review by Silvan-WMDE — APPROVED** [review]
I like it. If we have "Supported value change", should we then also include "Attempt to change unsupported value"? Or are the unhappy paths not worth creating diagrams?

**Comment by addshore** [comment]
> I like it. If we have "Supported value change", should we then also include "Attempt to change unsupported value"? Or are the unhappy paths not worth creating diagrams?

IMO the unhappy path doesn't really have much to do with architecture, as this all happens within the bridge itself, which is why I didn't include it in this PR.

**Comment by jakobw** [comment]
> I like it. If we have "Supported value change", should we then also include "Attempt to change unsupported value"? Or are the unhappy paths not worth creating diagrams?

What Adam said +

> Remark: The main criterion for the choice of possible scenarios (sequences, workflows) is their architectural relevancy. It is not important to describe a large number of scenarios. You should rather document a representative selection.

https://docs.arc42.org/section-6/ and https://docs.arc42.org/tips/6-2/

**Comment by addshore** [comment]
> > I like it. If we have "Supported value change", should we then also include "Attempt to change unsupported value"? Or are the unhappy paths not worth creating diagrams?
> 
> What Adam said +
> 
> > Remark: The main criterion for the choice of possible scenarios (sequences, workflows) is their architectural relevancy. It is not important to describe a large number of scenarios. You should rather document a representative selection.
> 
> https://docs.arc42.org/section-6/ and https://docs.arc42.org/tips/6-2/

I'm starting to think that we should create our own custom guide for arc42 that points to various tips and things.
Not sure where that would live though.

But I can see this as being a great example for how to sequence diagram.
We could compare this to the lower level sequence diagram that shows all of the things, and the fact that we should 2 key usecases in these docs, and link to the tipcs.

But perhaps that is too much. We don't want to write an arc42 guide after all

**Review by tzhelyazkova — APPROVED** [review]

**Pull request merged**